sw/snRuntime/src/start.c

```diff
@@ -84,12 +84,6 @@ static inline void snrt_init_cls() {
 
 static inline void snrt_init_libs() { snrt_alloc_init(); }
 
-static inline void snrt_exit(int exit_code) {
-    extern volatile uint32_t tohost;
-
-    if (snrt_global_core_idx() == 0) tohost = (exit_code << 1) | 1;
-}
-
```

Comment:
This has to stay, otherwise the call to it below will give an error at compile time.

Reply:
It still compiles (see CI). I moved it to [target/snitch_cluster/sw/runtime/rtl/src/snitch_cluster_start.c](https://github.com/pulp-platform/snitch_cluster/pull/31/files#diff-2989eedc7e2459373210b51f82afebfb9c02a0623ac4ccce51191b7aa9803aae). In that way it can be overwritten in the Banshee runtime with Banshee-specific return. Previously the two exit functions for RTL and Banshee had different names which prevented the OMP runtime (which explicitly calls `snrt_exit` in it's runtime part) from running on Banshee.